### PR TITLE
Add API for Articles and Resources

### PR DESCRIPTION
**Pull request by srri-zz**
Addition of:
- API endpoint for `articles` and `resources` models
- Added new requirement for mysqldb (python)
- Changed some imports to reflect the new structure

### CODE REVIEW

**Comment by srri-zz** [comment]
@psiemens or @blx, thoughts?

File: apps/api/serializers.py [hunk 1]
```diff
@@ -1,12 +1,24 @@
 __author__ = 'Steven Richards'
 from django.contrib.auth.models import Group
 from django.contrib.auth import get_user_model
+from dispatch.apps.content.models import Resource, Article
```
Comment (psiemens) [inline apps/api/serializers.py:4]: I believe this import statement can be simplified like so:

`from content.models import Resource, Article`

Django autoloads all the apps so you don't have to include the `dispatch.apps` part of the import statement, just the app name. Give it a try and see if it works.

**Comment by psiemens** [comment]
I don't think we need an API route for Resource because it's just a base model. Am I right, @blx?

The mysqldb requirement is a good call.

**Comment by blx** [comment]
Yeah it's abstract, I'm not sure it needs to be exposed in the API. As long as `Article` properly inherits the `Resource` fields.

Should `django` itself be in requirements? (If I make a new virtualenv and just install djangorestframework it doesn't pull down django.)

File: apps/api/serializers.py (hunk 1 above)
Comment (blx) [inline apps/api/serializers.py:4]: On the other hand, we could keep the fully qualified name to be explicit about what `content` or `core` etc are.

**Comment by psiemens** [comment]
Yeah, I guess it should be.

File: apps/api/serializers.py (hunk 1 above)
Comment (psiemens) [inline apps/api/serializers.py:4]: Yeah, that might be best. Good to avoid naming clashes with external libs

**Comment by srri-zz** [comment]
Fixed the route for resource, I wasn't sure what is was in terms of our modelling, thanks for clarifying. Also added django as a requirement. Merging since I've made those changes

**Pull request merged**